packages/bento-design-system/src/Table/Table.tsx

```diff
@@ -387,11 +394,11 @@ function ColumnHeader<D extends Record<string, unknown>>({
 
   return (
     <Box
-      className={clsx(lastLeftSticky && lastLeftStickyColumn, stickyHeaders && stickyColumnHeader)}
+      className={[lastLeftSticky && lastLeftStickyColumn, stickyHeaders && stickyColumnHeader]}
```

Comment:
not strictly related, but `clsx` is useless here since `Box` already uses it internally to merge the classes